pandas/core/computation/eval.py

```diff
@@ -160,7 +160,7 @@ def _check_for_locals(expr: str, stack_level: int, parser: str):
         )
 
     if at_top_of_stack or not_pandas_parser:
-        for toknum, tokval in tokenize_string(expr):
+        for toknum, tokval in _tokenize_string(expr):
```

Comment:
see my comments below, we don't want to use private functions

Reply:
What exactly do you mean with the comments below? Do you mean moving the functions to a new module?

Reply:
I mean don't use private functions (so don't privatize it in the first place)